server/graphql/v2/interface/Account.js

```diff
@@ -265,7 +265,7 @@ const accountFieldsDefinition = () => ({
     type: new GraphQLList(PaymentMethod),
     description: 'The list of payment methods that this account can use to pay for Orders',
     args: {
-      types: {
+      type: {
```

Comment:
Looks like we should remove this block entirely

Reply:
Ops, this is my fault; I did some mistake in the rebase 🤯 😑